OracleTSAM/samples/xe-db-svc/README.md

```diff
@@ -0,0 +1,42 @@
+Running Oracle TSAM Plus with oracle/database:11.2.0.2-xe image
+=============================
+You can run the TSAM container with the `oracle/database:11.2.0.2-xe` image provided on [GitHub](https://github.com/oracle/docker-images/tree/master/OracleDatabase).
+
+Due to the `oracle/database:11.2.0.2-xe` container creates new database and generates random `sys` user password on startup, a little "hacking" is needed here to make the `sys` password fixed.
+
+Basically this is done by overriding the default command `runOracle.sh` with this [modified one](cmd/runOracle.sh) in the `cmd` folder. The `docker-compose.yml` file to start the containers will look like below:
```

Comment:
@gvenzl Thank you for the comments. You are right, I also have a strong feeling now I need to rephrase those sentences.

Sorry for letting you confuse on the "hacking" word. I just wanted to express that replacing the database container's entry point command `runOracle.sh` with a modified one on its initialization to force using a designated password looks a little tricky.

The modified version of `runOracle.sh` actually just did a minor change to the original one by calling the `setPassword.sh` at near end of the script to set the SYS password to the one passed in by environment variable `DBA_PASSWD`.

Actually I also realized there could be some sync issue of this modified script with the base one in the future. While considering it's just provided as a sample and it's very easy for users to find out the changes by diffing the two, I think it would be okay to include it.

The intention of this change was to make the instantiation of the TSAM container along with its dependency could be done in one step (the `docker-compose up` command). Otherwise there will be at least 2 steps:
1. Run the database container with `docker run` command. Staring at the output or come back checking the result later until the initialization completes.
2. If a custom SYS password is desired, use `setPassword.sh` to change the password inside the database container.
3. Start the TSAM container and pass in the database connection string and credentials.

I think to eliminate all these efforts for overriding the default container command, or make it possible to use the database image in an automated environment, we may need to do some small enhancement to the database image. Do you think it's feasible for the database container to use a passed in SYS password on its initialization? For example, change line 58 of `runOracle.sh` to something like:
```bash
ORACLE_PWD=${ORACLE_PWD:-`openssl rand -hex 8`}
```
This will use the value of environment variable `ORACLE_PWD` as SYS password if it already exists, or else generate random password.